Templates/TypeScript/src/entity_types.ts.tt

```diff
@@ -9,45 +10,159 @@
     var entityTypes             = model.GetEntityTypes();
     var enumTypes               = model.GetEnumTypes();
     var complexTypes            = model.GetComplexTypes();
+
+    var maxLineLength = 180;
 #>
-// Type definitions for the Microsoft Graph <VERSION_STRING>
+// Type definitions for non-npm package microsoft-graph <VERSION_STRING>
```

Comment:
Where is `<VERSION_STRING>` used?

Reply:
Refer this comment in previous PR https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/203#discussion_r289249276